internal/controller/controller.go

```diff
@@ -127,14 +127,19 @@ func (controller *Controller) setupDatabase() error {
 		password := os.Getenv("DB_PASSWORD")
 
 		if uri == "" || username == "" || password == "" {
-			log.Error("you must set 'MONGODB_URI' or 'DB_USERNAME' or 'DB_PASSWORD' in the environment variable")
+			log.Error(
+				"you must set 'MONGODB_URI' or 'DB_USERNAME' or 'DB_PASSWORD' in the environment variable",
```

Comment:
It'd be nice to have formatting fixes to unrelated code in another PR or at least not in this one.